src/page-multi-facility/FacilityRow.tsx

```diff
@@ -35,6 +38,27 @@ const groupStatus = {
   infectious: true,
 };
 
+const pillTextColor: { [x: string]: string } = {
+  [RateOfSpreadType.CONTROLLED]: Colors.green,
+  [RateOfSpreadType.INFECTIOUS]: Colors.darkRed,
+  [RateOfSpreadType.MISSING]: Colors.forest,
+};
+
+const pillBorderColor: { [x: string]: string } = {
+  [RateOfSpreadType.CONTROLLED]: Colors.teal,
+  [RateOfSpreadType.INFECTIOUS]: Colors.orange,
+  [RateOfSpreadType.MISSING]: Colors.forest30,
+};
```

Comment:
looks like you either don't need these or they should be part of the pill component

Reply:
Yep - deleted.